Xcode/PDFImage/PDFImageView.m

```diff
@@ -97,6 +108,11 @@ - (void)willMoveToWindow:(UIWindow *)newWindow
 	self.options.scale = newWindow.screen.scale;
 }
 
+- (void)sizeToFit
```

Comment:
Please override sizeThatFits: rather than sizeToFit, the default implementation of sizeToFit in turn calls sizeThatFits: to adjust the bounds/frame